doc/crud.md

```diff
@@ -0,0 +1,106 @@
+# Access a document
+
+#### Request
+```http
+GET /data/:type/:id
+```
+```http
+GET /data/events/6494e0ac-dfcb-11e5-88c1-472e84a9cbee
+```
+
+#### Response OK
+```http
+HTTP/1.1 200 OK
+Date: Mon, 27 Sept 2016 12:28:53 GMT
+Content-Length: ...
+Content-Type: application/vnd.api+json
+Etag: "3-6494e0ac6494e0ac"
+```
+```json
+{
+  "data": {
+    "_id": "event/6494e0ac-dfcb-11e5-88c1-472e84a9cbee",
+    "_rev": "3-6494e0ac6494e0ac",
+    "_type": "events",
+    "startdate": "20160823T150000Z",
+    "enddate": "20160923T160000Z",
+    "summary": "A long month",
+    "description": "A could go on and on and on ....",    
+  }
+}
+```
+
+### Response Error
+```http
+HTTP/1.1 404 Not Found
+Content-Length: ...
+Content-Type: application/vnd.api+json
+```
+```json
+{
+  "errors": [ {
+    "code": "deleted",
+    "status": 404,
+    "title": "Event deleted",
+    "details": "Event 6494e0ac-dfcb-11e5-88c1-472e84a9cbee was deleted",
+    "links": {"about": "http://github.com/cozy/cozy-stack/docs/errors/deleted"}
+  }]
+}
+```
+
+### possible errors : 
+- 403 unauthenticated
+- 401 unauthorized
+- 404 deleted
+- 404 missing
+- 500 unkown
+
+--------------------------------------------------------------------------------
+
+# Create a document
+
+### Request
+```http
+POST /data/:type/(:id ?)
```

Comment:
I think we should have 2 distinct routes: POST /data/:type/ and PUT /data/:type/:id